Add srp_energy doc

briglx opened the pull request:
**Description:**
Add Documentation for Srp Energy.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27866

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

briglx committed:
Add documentation for Srp Energy

briglx committed:
Add documentation for Srp Energy

briglx committed:
Fix typo. Add standard configuration section.

briglx committed:
Fix typo. Add standard configuration section.

KaiboshOz committed:
username-host field - clarify use of '%' (#10347)

* username-host field - clarify use of '%'

Clarify that '%' is used in host field when wanting to connect *from* multiple hosts

* :pencil2: Tweak

xelprep committed:
Snips Doc - Fix link (#10772)

* Snips Doc - Fix link

The original author intended to link to a specific `if` statement that strips a username from an intent.

* :pencil2: Removed the link

calebmah committed:
Fix mqtt discovery link (#10858)

jjok committed:
Fix incorrect summary of services (#10860)

calebmah committed:
Fix broken links (#10855)

* fix generic camera link

* fix hassbian.local link

* fix mqtt discovery link

ketilmo committed:
Updated link to luci wiki page (#10886)

hmmbob committed:
🔗 Updating broken links in "oasa_telematics" (#10884)

hmmbob committed:
🔗 Updating broken link in "discord" (#10883)

ketilmo committed:
Added status for Samsung Q6F (#10879)

calebmah committed:
Fix blog link (#10876)

calebmah committed:
Fix websocket sensor link (#10877)

hmmbob committed:
🔗 Updating broken links in "Hass.IO installation" (#10882)

chriscla committed:
Fix Transmission example message formatting (#10875)

* Fix message formatting

* :pencil2: Tweak

calebmah committed:
Fix openalpr_local links (#10878)

hmmbob committed:
Updating broken links (#10880)

Replaced links to non-existing domains with code blocks.

claughinghouse committed:
Remove analytics-receiver source ref (#10870)

Ref #10491

hmmbob committed:
🔗 Fix broken local link (#10881)

ketilmo committed:
Updated http to https in URL (#10896)

ketilmo committed:
Updated http to https in URL (#10895)

ketilmo committed:
Updated http to https in URL (#10894)

ketilmo committed:
Updated http to https in link URL (#10893)

ketilmo committed:
Removed dead link (#10891)

ketilmo committed:
Updated Avion URL to https (#10890)

hmmbob committed:
🔗 Updating broken links in "habitica" and "html5" (#10889)

* 🔗 Updating broken links in "habitica"

* 🔗 Updating broken links in "HTML5"

ketilmo committed:
Removed dead link (#10887)

ketilmo committed:
Updated Jinja URLs (#10892)

a-r-j-a-n committed:
Update secrets.markdown (#10864)

Change because http password moved to auth providers.

basnijholt committed:
write water_heater in monospaced font (#10861)

Derkades committed:
Add instructions for upgrading  Home Assistant on FreeNAS (#10900)

StephanVinkenborg committed:
Propose adding test rights to usergroup. (#10859)

I got an error without the test rights that it couldn't perform a ping. Ping rights in mikrotik are in the test policy.

mrwacky42 committed:
nginx_proxy: Add documentation for `cloudflare` flag (#10857)

Partial fix for: https://github.com/home-assistant/hassio-addons/issues/776
See also: https://github.com/home-assistant/hassio-addons/pull/795

MBlokhuijzen committed:
Xiaomi Aqara Documentation: Add example of notifying user when… (#10846)



Co-authored-by: Melvin Blokhuijzen <mblokhuijzen@bol.com>

akasma74 committed:
time: relative_time function - description added (#10847)

It seems like the function itself has been there for a while but no mention of it in HA docs.

iantrich committed:
Lovelace: missing theme option for thermostat card (#10868)

iantrich committed:
Lovelace: missing theme option on light card (#10867)

elupus committed:
Remove pre 0.80 upgrade help (#10904)

peteclarkez committed:
Fix Client ID in setup instructions (#10903)

Hi 
I've just setup an alexa skill using these instructions.
All seemed to work well.

2 Issues
1. The Client ID for EU doesn't seem to be correct.
I've fixed it here.

2. Error linking lambda and skill

When attaching the lamdba reference to the Skill I came across the issue and solution below (found from the error msg)
https://forums.developer.amazon.com/questions/183361/failed-to-save-skill-information.html
I could add a note for that , but I'm assuming it's something local to what I did.

Hope it helps
Peter

briglx committed:
Merge remote-tracking branch 'upstream/next' into next

briglx committed:
Add srp_energy component.

briglx committed:
Add srp_energy component.

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas reviewed (changes requested):
Please rename the file to: `srpenergy.markdown` and move it to the `_integrations` folder

klaasnicolaas committed:
🚚 Move integrations to the right folder (#10909)

frenck committed:
Merge branch 'current' into next

engrbm87 committed:
Update glances.markdown (#10711)

roitto committed:
Add more info about bluestacks token method (#10853)

* Add more info about bluestacks token method

The bluestacks method didn't really tell enough how to get the token using the Bluestacks tweaker.

* Update vacuum.xiaomi_miio.markdown

Fix small typos.

pho3nixf1re committed:
rest_command now supports PATCH request method (#10913)

briglx committed:
Remove duplicate file. Remove link

briglx committed:
Merge with upstream

briglx commented:
@klaasnicolaas I created a new pull request for this #10920
